TestUtils/src/main/java/com/microsoft/device/dualscreen/testutils/ScreenshotComparator.kt

```diff
@@ -82,10 +82,6 @@ fun saveScreenshotToDevice(filename: String, node: SemanticsNodeInteraction) {
  */
 @RequiresApi(Build.VERSION_CODES.O)
 fun Bitmap.compare(other: Bitmap): Boolean {
-    if (this.width != other.width || this.height != other.height) {
-        throw AssertionError("Size of screenshot does not match golden file. Expected: $width $height Actual: ${other.width} ${other.height}")
```

Comment:
Maybe we can return false here instead, so we can keep the check but it doesn't cause an error?